qt5 5.14.2-1 CMake builds broken

```
CMake Error at D:/msys64/mingw64/qt5-static/lib/cmake/Qt5Core/Qt5CoreConfig.cmake:15 (message):
  The imported target "Qt5::Core" references the file

     "D:/msys64/mingw64/qt5-static/lib/libQt5Core.dll.a"

  but this file does not exist.  Possible reasons include:

  * The file was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and contained

     "D:/msys64/mingw64/qt5-static/lib/cmake/Qt5Core/Qt5CoreConfig.cmake"

  but not all the files it references.

Call Stack (most recent call first):
  D:/msys64/mingw64/qt5-static/lib/cmake/Qt5Core/Qt5CoreConfig.cmake:119 (_qt5_Core_check_file_exists)
  D:/msys64/mingw64/qt5-static/lib/cmake/Qt5Core/Qt5CoreConfig.cmake:320 (_populate_Core_target_properties)
  D:/msys64/mingw64/qt5-static/lib/cmake/Qt5Concurrent/Qt5ConcurrentConfig.cmake:223 (find_package)
  D:/msys64/mingw64/qt5-static/lib/cmake/Qt5/Qt5Config.cmake:28 (find_package)
  CMakeLists.txt:73 (find_package)
```

--- a/mingw-w64-qt5-static/0008-qt-5.3.0-win32-g-Add-QMAKE_EXTENSION_IMPORTLIB-defaulting-to-.patch
+++ b/mingw-w64-qt5-static/0008-qt-5.3.0-win32-g-Add-QMAKE_EXTENSION_IMPORTLIB-defaulting-to-.patch
@@ -10,23 +10,24 @@ diff -Naur qt-everywhere-src-5.12.4-orig/qtbase/mkspecs/common/g++-win32.conf qt
  
  PKG_CONFIG              = $${CROSS_COMPILE}pkg-config
  QMAKE_PKG_CONFIG        = $${CROSS_COMPILE}pkg-config
-diff -Naur qt-everywhere-src-5.12.4-orig/qtbase/mkspecs/features/create_cmake.prf qt-everywhere-src-5.12.4/qtbase/mkspecs/features/create_cmake.prf
---- qt-everywhere-src-5.12.4-orig/qtbase/mkspecs/features/create_cmake.prf	2019-06-12 23:59:14.000000000 +0300
-+++ qt-everywhere-src-5.12.4/qtbase/mkspecs/features/create_cmake.prf	2019-06-15 15:09:21.226485600 +0300
-@@ -324,11 +324,12 @@
-     }
- 
-     mingw {
--        CMAKE_WINMAIN_FILE_LOCATION_DEBUG = libqtmain$${QT_LIBINFIX}$${debug_suffix}.a
--        CMAKE_WINMAIN_FILE_LOCATION_RELEASE = libqtmain$${QT_LIBINFIX}.a
-+        isEmpty(QMAKE_EXTENSION_IMPORTLIB): QMAKE_EXTENSION_IMPORTLIB = dll.a
-+        CMAKE_WINMAIN_FILE_LOCATION_DEBUG = libqtmain$${QT_LIBINFIX}$${debug_suffix}.$${QMAKE_EXTENSION_IMPORTLIB}
-+        CMAKE_WINMAIN_FILE_LOCATION_RELEASE = libqtmain$${QT_LIBINFIX}.$${QMAKE_EXTENSION_IMPORTLIB}
- 
+diff --git a/mingw-w64-qt5-base/src/qtbase-everywhere-src-5.14.2/mkspecs/features/create_cmake.prf b/mingw-w64-qt5-base/src/qtbase-everywhere-src-5.14.2/mkspecs/features/create_cmake.prf
+index 0e71fd001..61d225310 100644
+--- a/qtbase/mkspecs/features/create_cmake.prf
++++ b/qtbase/mkspecs/features/create_cmake.prf
+@@ -328,8 +328,14 @@ mac {
+         CMAKE_WINMAIN_FILE_LOCATION_DEBUG = libqtmain$${QT_LIBINFIX}$${debug_suffix}.a
+         CMAKE_WINMAIN_FILE_LOCATION_RELEASE = libqtmain$${QT_LIBINFIX}.a
+
 -        CMAKE_IMPLIB_FILE_LOCATION_DEBUG = lib$${CMAKE_QT_STEM}$${debug_suffix}.a
 -        CMAKE_IMPLIB_FILE_LOCATION_RELEASE = lib$${CMAKE_QT_STEM}.a
-+        CMAKE_IMPLIB_FILE_LOCATION_DEBUG = lib$${CMAKE_QT_STEM}$${debug_suffix}.$${QMAKE_EXTENSION_IMPORTLIB}
-+        CMAKE_IMPLIB_FILE_LOCATION_RELEASE = lib$${CMAKE_QT_STEM}.$${QMAKE_EXTENSION_IMPORTLIB}
++        !isEmpty(CMAKE_STATIC_TYPE) {
++            CMAKE_IMPLIB_FILE_LOCATION_DEBUG = lib$${CMAKE_QT_STEM}$${debug_suffix}.a
++            CMAKE_IMPLIB_FILE_LOCATION_RELEASE = lib$${CMAKE_QT_STEM}.a
++        } else {
++            isEmpty(QMAKE_EXTENSION_IMPORTLIB): QMAKE_EXTENSION_IMPORTLIB = dll.a
++            CMAKE_IMPLIB_FILE_LOCATION_DEBUG = lib$${CMAKE_QT_STEM}$${debug_suffix}.$${QMAKE_EXTENSION_IMPORTLIB}
++            CMAKE_IMPLIB_FILE_LOCATION_RELEASE = lib$${CMAKE_QT_STEM}.$${QMAKE_EXTENSION_IMPORTLIB}
++        }
      } else {
          CMAKE_WINMAIN_FILE_LOCATION_DEBUG = qtmain$${QT_LIBINFIX}$${debug_suffix}.lib
          CMAKE_WINMAIN_FILE_LOCATION_RELEASE = qtmain$${QT_LIBINFIX}.lib

--- a/mingw-w64-qt5-static/PKGBUILD
+++ b/mingw-w64-qt5-static/PKGBUILD
@@ -112,7 +112,7 @@ _ver_base=5.14.2
 # use 5.6.1-1 hot fix : only the archive name was changed from *-5.6.1.tar.xz to *-5.6.1-1.tar.xz
 _hotfix=
 pkgver=${_ver_base//-/}
-pkgrel=1
+pkgrel=2
 arch=('any')
 pkgdesc="A cross-platform application and UI framework (mingw-w64${_namesuff})"
 url='https://www.qt.io/'
@@ -789,7 +789,7 @@ sha256sums=('c6fcd53c744df89e7d3223c02838a33309bd1c291fcb6f9341505fe99f7f19fa'
             'e04033364f5ad77906b4100c34572df57bd793a55bc33b7d8a753a2cc60af259'
             '18fd2fa42215ac47b3b314261ab98cbe65f8231429e4f29a152288a3ca93daf1'
             '4e154fbc9059a096c351d019da6b18c907b1d8b06e028f48c7365f62bcd0edc9'
-            '1efa0f4abc19a1544ff0613bea318043d26c73cb0636bcfc2066cd1012c394d7'
+            'd99a4b74a062912a6913a883655d67257715866209a5c17cf9a3e6837dbd9260'
             '465d3897c4494695b66ac121944b3bd370db91bfea71dd2c9170dbf268cc99ea'
             '678c185795a41c7f38822120030585a0427287f3a1c2df9616daaa89f3285df3'
             '61175144833ff17d60018b31c6434afce3adc729a13d0d21021adec3be191185'

--- a/mingw-w64-qt5/0008-qt-5.3.0-win32-g-Add-QMAKE_EXTENSION_IMPORTLIB-defaulting-to-.patch
+++ b/mingw-w64-qt5/0008-qt-5.3.0-win32-g-Add-QMAKE_EXTENSION_IMPORTLIB-defaulting-to-.patch
@@ -10,23 +10,24 @@ diff -Naur qt-everywhere-src-5.12.4-orig/qtbase/mkspecs/common/g++-win32.conf qt
  
  PKG_CONFIG              = $${CROSS_COMPILE}pkg-config
  QMAKE_PKG_CONFIG        = $${CROSS_COMPILE}pkg-config
-diff -Naur qt-everywhere-src-5.12.4-orig/qtbase/mkspecs/features/create_cmake.prf qt-everywhere-src-5.12.4/qtbase/mkspecs/features/create_cmake.prf
---- qt-everywhere-src-5.12.4-orig/qtbase/mkspecs/features/create_cmake.prf	2019-06-12 23:59:14.000000000 +0300
-+++ qt-everywhere-src-5.12.4/qtbase/mkspecs/features/create_cmake.prf	2019-06-15 15:09:21.226485600 +0300
-@@ -324,11 +324,12 @@
-     }
- 
-     mingw {
--        CMAKE_WINMAIN_FILE_LOCATION_DEBUG = libqtmain$${QT_LIBINFIX}$${debug_suffix}.a
--        CMAKE_WINMAIN_FILE_LOCATION_RELEASE = libqtmain$${QT_LIBINFIX}.a
-+        isEmpty(QMAKE_EXTENSION_IMPORTLIB): QMAKE_EXTENSION_IMPORTLIB = dll.a
-+        CMAKE_WINMAIN_FILE_LOCATION_DEBUG = libqtmain$${QT_LIBINFIX}$${debug_suffix}.$${QMAKE_EXTENSION_IMPORTLIB}
-+        CMAKE_WINMAIN_FILE_LOCATION_RELEASE = libqtmain$${QT_LIBINFIX}.$${QMAKE_EXTENSION_IMPORTLIB}
- 
+diff --git a/mingw-w64-qt5-base/src/qtbase-everywhere-src-5.14.2/mkspecs/features/create_cmake.prf b/mingw-w64-qt5-base/src/qtbase-everywhere-src-5.14.2/mkspecs/features/create_cmake.prf
+index 0e71fd001..61d225310 100644
+--- a/qtbase/mkspecs/features/create_cmake.prf
++++ b/qtbase/mkspecs/features/create_cmake.prf
+@@ -328,8 +328,14 @@ mac {
+         CMAKE_WINMAIN_FILE_LOCATION_DEBUG = libqtmain$${QT_LIBINFIX}$${debug_suffix}.a
+         CMAKE_WINMAIN_FILE_LOCATION_RELEASE = libqtmain$${QT_LIBINFIX}.a
+
 -        CMAKE_IMPLIB_FILE_LOCATION_DEBUG = lib$${CMAKE_QT_STEM}$${debug_suffix}.a
 -        CMAKE_IMPLIB_FILE_LOCATION_RELEASE = lib$${CMAKE_QT_STEM}.a
-+        CMAKE_IMPLIB_FILE_LOCATION_DEBUG = lib$${CMAKE_QT_STEM}$${debug_suffix}.$${QMAKE_EXTENSION_IMPORTLIB}
-+        CMAKE_IMPLIB_FILE_LOCATION_RELEASE = lib$${CMAKE_QT_STEM}.$${QMAKE_EXTENSION_IMPORTLIB}
++        !isEmpty(CMAKE_STATIC_TYPE) {
++            CMAKE_IMPLIB_FILE_LOCATION_DEBUG = lib$${CMAKE_QT_STEM}$${debug_suffix}.a
++            CMAKE_IMPLIB_FILE_LOCATION_RELEASE = lib$${CMAKE_QT_STEM}.a
++        } else {
++            isEmpty(QMAKE_EXTENSION_IMPORTLIB): QMAKE_EXTENSION_IMPORTLIB = dll.a
++            CMAKE_IMPLIB_FILE_LOCATION_DEBUG = lib$${CMAKE_QT_STEM}$${debug_suffix}.$${QMAKE_EXTENSION_IMPORTLIB}
++            CMAKE_IMPLIB_FILE_LOCATION_RELEASE = lib$${CMAKE_QT_STEM}.$${QMAKE_EXTENSION_IMPORTLIB}
++        }
      } else {
          CMAKE_WINMAIN_FILE_LOCATION_DEBUG = qtmain$${QT_LIBINFIX}$${debug_suffix}.lib
          CMAKE_WINMAIN_FILE_LOCATION_RELEASE = qtmain$${QT_LIBINFIX}.lib

--- a/mingw-w64-qt5/PKGBUILD
+++ b/mingw-w64-qt5/PKGBUILD
@@ -112,7 +112,7 @@ _ver_base=5.14.2
 # use 5.6.1-1 hot fix : only the archive name was changed from *-5.6.1.tar.xz to *-5.6.1-1.tar.xz
 _hotfix=
 pkgver=${_ver_base//-/}
-pkgrel=1
+pkgrel=2
 arch=('any')
 pkgdesc="A cross-platform application and UI framework (mingw-w64${_namesuff})"
 url='https://www.qt.io/'
@@ -789,7 +789,7 @@ sha256sums=('c6fcd53c744df89e7d3223c02838a33309bd1c291fcb6f9341505fe99f7f19fa'
             'e04033364f5ad77906b4100c34572df57bd793a55bc33b7d8a753a2cc60af259'
             '18fd2fa42215ac47b3b314261ab98cbe65f8231429e4f29a152288a3ca93daf1'
             '4e154fbc9059a096c351d019da6b18c907b1d8b06e028f48c7365f62bcd0edc9'
-            '1efa0f4abc19a1544ff0613bea318043d26c73cb0636bcfc2066cd1012c394d7'
+            'd99a4b74a062912a6913a883655d67257715866209a5c17cf9a3e6837dbd9260'
             '465d3897c4494695b66ac121944b3bd370db91bfea71dd2c9170dbf268cc99ea'
             '678c185795a41c7f38822120030585a0427287f3a1c2df9616daaa89f3285df3'
             '61175144833ff17d60018b31c6434afce3adc729a13d0d21021adec3be191185'

